src/routes/auth.js

```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import { verifyToken } from '../utils/utils';
+import { tokenName, rememberTokenName } from '../config/keys';
 
 const router = express.Router();
 
 // post route to check validity of tokens, clients will hit this route.
-router.post('/verify-token', (req, res) => {
-    // extract token from post request headers
-    const token = req.header('access-token');
-    const rememberme = req.header('refresh-token');
+router.post('/refresh-token', (req, res) => {
+    // Extract tokens from request body
+    const token = req.body[tokenName];
+    const rememberme = req.body[rememberTokenName];
```

Comment:
we extract token from `req.header`, not `req.body`

Reply:
And I think naming convention, `access-token-name` and `refresh-token-name` is better.

Reply:
Also change every instance of `rememberme` variable to `refresh_token` maybe

Reply:
> we extract token from `req.header`, not `req.body`
@bicycleman15 
Afaik, the data in a POST request is sent in the body and not the headers. Will sending it in the headers have any advantage over sending it in the body?

Reply:
It's a convention to send token in header

Reply:
Have a look at this. https://stackoverflow.com/questions/53880867/is-there-any-security-difference-between-http-headers-and-http-body

Reply:
Yeah I understand, that is the case when you need to authenticate with an API endpoint, then we do send our authorization token in the `Authorization` header. The API endpoint then authorizes the **sender of the request** and then allows access to data that the endpoint is supposed to provide/operate upon. In our case the endpoint's task itself is to verify the token and the remember-me token. 

So in our example suppose we need to authenticate the SSO client (which we have not implemented) like yearbook, then we will provide an access token to yearbook which it will send in the POST request with `Authorization: Bearer <yearbook_token>`. and the tokens (`token` and `rememberme`) that need to be verified for the end user visiting yearbook will be sent in the body.

So basically what I am trying to say is that in our case the token that we are sending to the API endpoint **are NOT the tokens that authorize the sender of the request (yearbook)** to access the endpoint, but they themselves are the data that the endpoint is supposed to operate on.

I hope you get my point.

Reply:
Okay then, if you insist.